src/test/java/io/jenkins/plugins/grading/TestScoreTest.java

```diff
@@ -48,4 +51,15 @@ void shouldCalculateNegativeResult() {
 
         assertThat(test.getTotalImpact()).isEqualTo(-9);
     }
+
+    @Test
+    void shouldConvertFromJson() {
+        TestConfiguration configuration = TestConfiguration.from(JSONObject.fromObject(
+                "{\"maxScore\":5,\"failureImpact\":1,\"passedImpact\":2,\"skippedImpact\":3}"));
+
+        assertThat(configuration.getMaxScore()).isEqualTo(5);
```

Comment:
AssertJ Custom Assertions nutzen:
```suggestion
        assertThat(configuration).hasMaxScore(5);
```